### PR TITLE
Fix #201

### LINKED ISSUE #201: Does not work with currency symbols longer than 3 chars
Such as TESTBTC or TESTUSD. It applies also to perpetual swaps.

### PR DESCRIPTION
**Pull request by hhanh00**

### CODE REVIEW

File: src/main/java/com/github/jnidzwetzki/bitfinex/v2/entity/currency/BitfinexCurrencyPair.java [hunk 1]
```diff
@@ -255,7 +255,7 @@ public static BitfinexCurrencyPair fromSymbolString(final String symbolString) {
 	 */
 	@Override
 	public String toBitfinexString() {
-		if(currencyType == BitfinexCurrencyType.PERPETUAL) {
+		if (currency1.length() > 3 || currency2.length() > 3) {
```
Comment (jnidzwetzki) [inline src/main/java/com/github/jnidzwetzki/bitfinex/v2/entity/currency/BitfinexCurrencyPair.java:258]: Could you add this logic to the parseCurrencyType() method and set `BitfinexCurrencyType.PERPETUAL` in this case?

**Review by jnidzwetzki — CHANGES_REQUESTED** [review]
Hi,

Thank you very much for your contribution to this project. Might it be possible to move the adjustment to the `parsePair()` method? You could set the `BitfinexCurrencyType` to `PERPETUAL` in this method directly. 

Thank you very much
   Jan

**Comment by hhanh00** [comment]
I'm not sure I understand what you mean. The serialization of a currency pair with ':' does not seem related to its type. For example TESTBTC:TESTUSD is a regular pair.

**Comment by jnidzwetzki** [comment]
You are right. I thought `TESTBTC:TESTUSD` is also a PERPETUAL. 

Your PR is fine, thank you for your contribution.

**Pull request merged**